exec-healthz/Makefile

```diff
@@ -1,17 +1,69 @@
-all: push
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
-# 0.0 shouldn't clobber any released builds
-TAG = 0.0
-PREFIX = gcr.io/google_containers/exechealthz
+# Build the exechealthz binary or image for any GOARCH
+#
+# Usage:
+# 	[TAG=1.0] [REGISTRY=gcr.io/google_containers] [ARCH=amd64] make [server|container|push]
```

Comment:
Yeah please change to PREFIX, because it doesn't have to be a registry and if I wrote a build script that rebuilds all images in contrib it would be nice to not have to special case this one.